include/lemlib/movements/purepursuit.hpp

```diff
@@ -45,17 +45,17 @@ class PurePursuit : public Movement {
         std::pair<int, int> update(Pose pose) override;
     private:
         std::vector<Waypoint> path;
-        Pose prevPose = Pose(0, 0, 0);
-        Waypoint prevLookahead = Waypoint(0, 0);
-        float trackWidth;
-        int startTime;
-        float lookaheadDist;
-        int timeout;
+        Pose prevPose = Pose(0_in, 0_m, 0_rad);
+        Waypoint prevLookahead = Waypoint(0_in, 0_m);
```

Comment:
```suggestion
        Waypoint prevLookahead = Waypoint(0_m, 0_m);
```
Internal units should be consistent